src/desktop/toolwidgets/fillsettings.cpp

```diff
@@ -13,7 +13,7 @@ namespace tools {
 
 namespace props {
 static const ToolProperties::RangedValue<int> expand{
-	QStringLiteral("expand"), 0, 0, 100},
+	QStringLiteral("expand"), -30, -30, 30},
```

Comment:
The default value should remain 0. Negative values only make sense if the rest of #1188 is actually implemented.